extensions/wikia/TemplateClassification/Permissions.class.php

```diff
@@ -15,8 +15,18 @@ class Permissions {
 	 * @return bool
 	 */
 	public function shouldDisplayEntryPoint( \User $user, \Title $title ) {
+		return $this->userCanChangeType( $user, $title )
+		       && $title->inNamespace( NS_TEMPLATE );
```

Comment:
I think you could leave ampersand characters in the line above - see https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#conditional-logic